server/neptune/workflows/activities/command/command_builder.go

```diff
@@ -0,0 +1,39 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"syscall"
+
+	"github.com/hashicorp/go-version"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/runtime/cache"
+)
+
+type execCmdBuilder struct {
```

Comment:
we know it's building an exec.Cmd based on the return type, it's weird having the package name and the struct in the name. even just builder is fine, or like execBuilder even since command is in the package name.